src/steps/01-AcquisitionPackageDetails/COR_ACOR/ContactInfoForm.vue

```diff
@@ -25,6 +26,7 @@
       :selectedValue.sync="_selectedBranch"
       :showAccessRadioButtons.sync="showAccessRadioButtons"
       :returnObject="true"
+      :rules="[$validators.required('Please select your ' + corOrAcor + '’s service branch.')]"
```

Comment:
Validation isn't triggering for some reason on this dropdown when tab in and tab out